packages/mask/src/social-network-adaptor/twitter.com/automation/pasteTextToComposition.ts

```diff
@@ -20,7 +20,9 @@ export const pasteTextToCompositionTwitter: SocialNetworkUI.AutomationCapabiliti
                 if (abort.aborted) throw new Error('Aborted')
             }
 
-            if (!isCompose() && !hasEditor() && opt?.reason !== 'reply') {
+            // Workaround: To open the compose dialog even if it's already opened(UX insensitivity)
+            //  to set the editor element as the correct `document.activeElement`.
+            if (opt?.reason !== 'reply') {
```

Comment:
Can we update the `isCompose()` and `hasEditor()` to match your situation? They must exist for some reasons.

Reply:
It's by intention.